inc/integrations/multilingualpress/namespace.php

```diff
@@ -0,0 +1,110 @@
+<?php
+/**
+ * Integration with MultilingualPress 3.
+ *
+ * @package asset-manager-framework
+ */
+
+declare( strict_types=1 );
+
+namespace AssetManagerFramework\Integrations\MultilingualPress;
+
+use Inpsyde\MultilingualPress\ {
+	TranslationUi\Post\RelationshipContext,
+	TranslationUi\Post\PostRelationSaveHelper,
+	Framework\Http\PhpServerRequest,
+};
+
+/**
+ * Bootstrap actions and filters.
+ */
+function bootstrap() : void {
+	add_filter( PostRelationSaveHelper::FILTER_SYNC_KEYS, __NAMESPACE__ . '\sync_thumbnail', 10, 3 );
```

Comment:
```suggestion
	add_filter( PostRelationSaveHelper::FILTER_SYNC_KEYS, __NAMESPACE__ . '\\sync_thumbnail', 10, 3 );
```
It is very important to realize that because the backslash is used as an escape character within strings, it should always be doubled when used inside a string. Otherwise there is a risk of unintended consequences using \n \r etc. https://www.php.net/manual/zh/language.namespaces.faq.php#language.namespaces.faq.quote